configs/kintohub.json

```diff
@@ -1,8 +1,7 @@
 {
   "index_name": "kintohub",
   "start_urls": [
-    "https://docs.kintohub.com/docs/getting-started.html",
-    "https://docs.kintohub.com/docs/"
+    "https://docs.kintohub.com/docs/introduction/getting-started"
```

Comment:
https://docs.kintohub.com/docs/introduction/getting-started will be the only page crawled and thus available from the search.

Reply:
thank you for the advice. and i added the sitemap as well.